src/toil/jobStores/googleJobStore.py

```diff
@@ -158,17 +158,26 @@ def destroy(self):
         except exceptions.NotFound:
             # just return if not connect to physical storage. Needed for idempotency
             return
+        
         try:
             self.bucket.delete(force=True)
-            # throws ValueError if bucket has more than 256 objects. Then we must delete manually
         except ValueError:
-            self.bucket.delete_blobs(self.bucket.list_blobs())
```

Comment:
Is composing the batch yourself here going to be more efficient than letting `delete_blobs` do it? Is `delete_blobs` documented not to use a batch internally, or do you have some other evidence that it doesn't?

If so, I'd add a comment here pointing to the evidence that led you to choose this approach over the more obvious single `delete_blobs` call.